automation/execute-tests.sh

```diff
@@ -23,15 +23,18 @@ echo "Using: "
 echo "  KUBECTL: $KUBECTL"
 echo "  KUBECONFIG: $KUBECONFIG"
 echo "  IMAGEIO_NAMESPACE: $IMAGEIO_NAMESPACE"
+echo "  VCSIM_NAMESPACE: $VCSIM_NAMESPACE"
 echo "  DEFAULT_SC: $DEFAULT_SC"
 echo "  NFS_SC: $NFS_SC"
 
 ./cluster/imageio-install.sh "$IMAGEIO_NAMESPACE"
+./cluster/vcsim-install.sh "$VCSIM_NAMESPACE"
 
 FAKEOVIRT_CA_PATH=${FAKEOVIRT_CA_PATH:-$(pwd)/_out/fakeovirt-ca.pem}
 
 $KUBECTL -n "$IMAGEIO_NAMESPACE" exec deploy/imageio-deployment -c imageiotest -- cat /tmp/certs/ca.pem > "$FAKEOVIRT_CA_PATH"
 
 $KUBECTL apply -f tests/os-mapping/os-mapping.yaml
 
-go test ./tests/ovirt --v -timeout 120m -kubeconfig "$KUBECONFIG" -ovirt-ca "$FAKEOVIRT_CA_PATH" -imageio-namespace "$IMAGEIO_NAMESPACE" -kubevirt-namespace "$KUBEVIRT_NAMESPACE" -default-sc "$DEFAULT_SC" -nfs-sc "$NFS_SC"
\ No newline at end of file
+go test ./tests/ovirt --v -timeout 120m -provider ovirt -kubeconfig "$KUBECONFIG" -ovirt-ca "$FAKEOVIRT_CA_PATH" -imageio-namespace "$IMAGEIO_NAMESPACE" -kubevirt-namespace "$KUBEVIRT_NAMESPACE" -default-sc "$DEFAULT_SC" -nfs-sc "$NFS_SC"
+go test ./tests/vmware --v -timeout 120m -provider vmware -kubeconfig "$KUBECONFIG" -kubevirt-namespace "$KUBEVIRT_NAMESPACE" -default-sc "$DEFAULT_SC" -nfs-sc "$NFS_SC"
```

Comment:
If the tests for oVirt fail and for VMware don't nobody will notice unless one looks into the prow logs; like here: https://prow.apps.ovirt.org/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_vm-import-operator/388/pull-vm-import-operator-e2e-k8s-1.18/1303949338703040512. Would it behave correctly if we run `go test ./tests --v ...`?

Reply:
I've pushed a new commit to make the provider flag unnecessary on the command line, and to run both sets of tests in the same command with `go test ./tests/ovirt ./tests/vmware`. That looks like it should work.